### PR TITLE
cargo-rustc: add page

### PR DESCRIPTION
**Pull request by navarroaxel**
- [x] The page (if new), does not already exist in the repo.
- [x] The page is in the correct platform directory (`common/`, `linux/`, etc.)
- [x] The page has 8 or fewer examples.
- [x] The PR title conforms to the recommended [templates](/tldr-pages/tldr/blob/master/CONTRIBUTING.md#commit-message).
- [x] The page follows the [content guidelines](/tldr-pages/tldr/blob/master/CONTRIBUTING.md#guidelines).
- [x] The page description includes a link to documentation or a homepage (if applicable).

### CODE REVIEW

File: pages/common/cargo-rustc.md [hunk 1]
```diff
@@ -0,0 +1,36 @@
+# cargo rustc
+
+> Compile a package, and pass extra options to the compiler.
+> More information: <https://doc.rust-lang.org/cargo/commands/cargo-rustc.html>.
+
+- Build the package or packages defined by the `Cargo.toml` manifest file in the local path:
```
Comment (mebeim) [inline pages/common/cargo-rustc.md:6]:
```suggestion
- Build the package or packages defined by the `Cargo.toml` manifest file in the current working directory:
```

File: pages/common/cargo-rustc.md [hunk 2]
```diff
@@ -0,0 +1,36 @@
...
+
+- Compile with architecture-specific optimizations for the current CPU:
+
+`cargo rustc --release -- -C target-cpu=native`
+
+- Compile with speed optimization:
+
+`cargo rustc -- -C opt-level {{1|2|3}}`
+
+- Compile with size optimization:
```
Comment (mebeim) [inline pages/common/cargo-rustc.md:22]:
```suggestion
- Compile with [s]ize optimization (`z` also turns off loop vectorization):
```

File: pages/common/cargo-rustc.md [hunk 3]
```diff
@@ -0,0 +1,36 @@
+# cargo rustc
+
+> Compile a package, and pass extra options to the compiler.
```
Comment (mebeim) [inline pages/common/cargo-rustc.md:3]:
```suggestion
> Compile a Rust package, and pass extra options to the compiler.
```

**Review by mebeim — CHANGES_REQUESTED** [review]
Thank you for the page @navarroaxel :) just some minor nitpicks below.

**Review by mebeim — APPROVED** [review]

**Review by sbrl — APPROVED** [review]
Thanks, @navarroaxel!

**Pull request merged**